rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -249,24 +250,19 @@ public class RskContext implements NodeContext, NodeBootstrapper {
     private PeerScoringReporterService peerScoringReporterService;
     private TxQuotaChecker txQuotaChecker;
     private GasPriceTracker gasPriceTracker;
-
     private volatile boolean closed;
+    private boolean versionOrHelpRequested;
```

Comment:
just an idea.. `versionOrHelpRequested` seems to be transitory and sort of responsibility of `CliArgs` class. Wdyt about extending NodeCliFlags with two new enums - VERSION and HELP - and populate with those the `CliArgs` collection if needed, so that it'd be consistent with other cli options. And if you wanna check if any of those was requested in a command, you can do `rskContent.getCliArgs().getFlags().containts()`

Reply:
Agree. Now I'm doing:

`if (ctx.getCliArgs().getFlags().contains(NodeCliFlags.VERSION) || ctx.getCliArgs().getFlags().contains(NodeCliFlags.HELP)) {
            return;
        }
        `
 don't you think it would be good to have a wrapper on RskContext like:
 `
 public boolean isVersionOrHelpRequested() {
       return ctx.getCliArgs().getFlags().contains(NodeCliFlags.VERSION) || ctx.getCliArgs().getFlags().contains(NodeCliFlags.HELP);
}
 `
 
 So that Start.main() looks a bit cleaner? I'm open to staying like this, just wondering WDYT.